content/en/getting_started/integrations/google_cloud.md

```diff
@@ -0,0 +1,339 @@
+| [Cloud Load Balancing][14]          | Distribute load-balanced compute resources                                            |
+| [Cloud Logging][15]                 | Real-time log management and analysis                                                 |
+| [Cloud Memorystore for Redis][16]   | A fully managed in-memory data store service                                          |
+| [Cloud Router][17]                  | Exchange routes between your VPC and on-premises networks by using BGP                |
+| [Cloud Run][18]                     | Managed compute platform that runs stateless containers through HTTP                  |
+| [Cloud Security Command Center][19] | Security Command Center is a threat reporting service.                                |
+| [Cloud Tasks][20]                   | Distributed task queues                                                               |
+| [Cloud TPU][21]                     | Train and run machine learning models                                                 |
+| [Compute Engine][22]                | High performance virtual machines                                                     |
+| [Container Engine][23]              | Kubernetes, managed by google                                                         |
+| [Datastore][24]                     | NoSQL database                                                                        |
+| [Firebase][25]                      | Mobile platform for application development                                           |
+| [Functions][26]                     | Serverless platform for building event-based microservices                            |
+| [Kubernetes Engine][27]             | Cluster manager and orchestration system                                              |
+| [Machine Learning][28]              | Machine learning services                                                             |
+| [Private Service Connect][29]       | Access managed services with private VPC connections                                  |
+| [Pub/Sub][30]                       | Real-time messaging service                                                           |
+| [Spanner][31]                       | Horizontally scalable, globally consistent, relational database service               |
+| [Storage][32]                       | Unified object storage                                                                |
+| [Vertex AI][33]                     | Build, train and deploy custom machine learning (ML) models.                          |
+| [VPN][34]                           | Managed network functionality                                                         |
+{{% /collapse-content %}} 
+
+You can find your Google Cloud metrics in the Metrics Summary page in the Datadog platform.
+To view your metrics, navigate to `Metrics > Summary` and search for `gcp`:
```

Comment:
```suggestion
To view your metrics, use the left menu to navigate to **Metrics** > **Summary** and search for `gcp`:
```
I recommend removing the image showing the navigation to the metrics summary. The navigation changes all the time, so screenshots of it quickly get out of date. The text description should be enough for customers to find the page.